neurom/fst/sectionfunc.py

```diff
@@ -72,6 +72,26 @@ def branch_order(section):
     return sum(1 for _ in section.iupstream()) - 1
 
 
+def strahler_order(section):
+    '''Branching order of a tree section
+
+    The strahler order is the inverse of the branch order,
+    since this is computed from the tips of the tree
+    towards the root.
+    This is computed as the maximum branch order of
+    a section's descendants minus the section's own branch order
+    plus one to start counting from 1.
+
+    Note:
+        The leaves have branch order 1.
+    '''
+    if len(section.children) == 0:
+        return 1
+    else:
+        max_bo_children = max([branch_order(i) for i in section.ipreorder()])
+        return max_bo_children - branch_order(section) + 1
```

Comment:
I am sure but I am not aware of a way since I haven't found any doc for this. For the neurons I tested it takes almost the same time as branch order so it wasn't an issue. Maybe add it as a issue to optimize this in the future?